addons/web/static/src/core/assets.js

```diff
@@ -99,7 +99,7 @@ const loadCSS = memoize(function loadCSS(url) {
  * @returns {Promise<XMLDocument>} A Promise of an XML document containing the
  *      owl templates.
  */
-const loadBundleTemplates = memoize(async function loadBundleTemplates(name) {
+export const loadBundleTemplates = memoize(async function loadBundleTemplates(name) {
```

Comment:
can't you use loadBundle instead?

Reply:
Let's talk about the API on monday. Not a fan of using a very generic API like loadBundle when you only want to load the templates, although in this case we would like to load the whole bundle, but loadBundle currently only allows to load templates (but with a clunkier API)